allure-report-face/src/styles.css

```diff
@@ -10,11 +11,13 @@
 @import "./blocks/link/styles.css";
 @import "./blocks/spinner/styles.css";
 @import "./blocks/text/styles.css";
+@import "./blocks/widget/styles.css";
 @import "./variables.css";
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,700&subset=latin,cyrillic);
 
 body {
   color: $text-color;
-  font-family: -apple-system, Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
```

Comment:
It means nothing. People would be just starring on white screen until this webfont will be fully loaded